README.md

```diff
@@ -84,6 +84,7 @@ Much of the philosophy behind Bedrock is inspired by the [Twelve-Factor App](htt
   - `DB_HOST` - Database host
   - Optionally, you can define `DATABASE_URL` for using a DSN instead of using the variables above (e.g. `mysql://user:password@127.0.0.1:3306/db_name`)
 - `WP_ENV` - Set to environment (`development`, `staging`, `production`)
+- `WP_ENVIRONMENT_TYPE` - Set to [environment type](https://developer.wordpress.org/reference/functions/wp_get_environment_type/)
```

Comment:
Let's omit this for now since by default it will be set automatically (and in ~98% of cases?). Don't want people thinking they need to set it.

Reply:
@swalkinshaw , ok. I've removed that line from README.